discovery-provider/src/queries/get_undisbursed_challenges.py

```diff
@@ -64,8 +62,6 @@ def get_undisbursed_challenges(
             Challenge.id == UserChallenge.challenge_id,
         )
         .join(User, UserChallenge.user_id == User.user_id)
-        # Join against UserBank to ensure only users with banks are disbursable
```

Comment:
What are the ramifications of removing this join if it was intended to only allow users with banks to receive challenge disbursements? Guessing at it, I could imagine:
* Failure gets pushed down to the claim call, which won't work until users have a bank. (Should we make sure that endpoint checks this? 😅 )
* This opens us up to some type of abuse? Though I can't imagine how, maybe you can.

[Looks like you added it originally](https://github.com/AudiusProject/audius-protocol/commit/cc902d8127e753d0b7ae4414305f7695c94f1076#diff-789fb0946d9e60220c3b8314d41fb79c164d7370a49851b1c717f1568ddf7d1bR69), so I trust you to know if it's safe to remove!

Reply:
Was about to write the same thing. Iirc with lazy user banks, the user bank is created when the user tries to claim a challenge for the first time right? So if we fail out there first we shouldn't reach this code.

Reply:
Re: abuse I'm trying to think, if the attacker can mimic a real claim request, but swap it out for a non-userbank wallet, that would go through now right? However they wouldn't be able to get any more $AUDIO than they would have been able to claim in the first place, they could just redirect the claim which is pointless.

Reply:
Great questions. A user can try to disburse without a userbank, but the *solana program* itself will prevent this from working on some arbitrary account. Here:

https://github.com/AudiusProject/audius-protocol/blob/ad66799a5d469df4a4511a4491d62936d4406981/solana-programs/reward-manager/program/src/processor.rs#L427

It asserts that the userbank is derived from the eth account, and DN quorum enforces a real eth address.

Reply:
Also @schottra a little historical context - the reason this join was there initially wasn't so much a security thing as fixing a bug where some users would fail to have userbanks created due to Solana congestion, and then our Identity Attester would try to attest for them and get stuck because the userbank didn't exist. However, now our Identity Attester is smart and will account for this!